src/entity-generator/entity-generator.cpp

```diff
@@ -200,25 +237,30 @@ void parseCommandLine(int argc, char *argv[]) {
       "Number of entities to generate topologies for")(
       "days", po::value<int>(&options.daysToRun)->default_value(1),
       "Days of topologies to send to pool server")(
-      "initialize-topology",
-      po::value<bool>(&options.initialize)->default_value(true),
-      "Add topologies to sluice.  Disable to only send topology updates.")(
       "center-start",
       po::value<bool>(&options.centerStart)->default_value(false),
       "Originate all nodes at the center of the United States.")(
       "march-west", po::value<bool>(&options.marchWest)->default_value(false),
       "Cause all nodes to move westward every interval")(
-      "live", po::value<bool>(&options.live)->default_value(false),
+      "live", po::bool_switch(&options.live)->default_value(false),
       "Generate topology updates now rather than at a fixed start date and "
       "interval")(
       "step-size", po::value<double>(&options.stepSize)->default_value(0.1),
       "Distance to move nodes every time interval (decimal degrees)")(
-      "time-interval", po::value<int>(&options.timeInterval)->default_value(15),
+      "time-interval", po::value<int>(&options.timeInterval)->default_value(1),
       "Seconds between topology updates")(
+      "endtime-offset",
+      po::value<uint64_t>(&options.endtimeOffset)->default_value(0),
+      "Duration after which entity expires (ms)")(
+      "start-time", po::value<uint64_t>(&options.startTime)->default_value(0),
```

Comment:
I'm not in love with this new setting.  I often enjoy the fact that entity generator defaults to current time.  It's a useful way to append data to an existing dataset.  For instance, when doing live tardis testing, I could create a single dataset/lens, set up my orchestration, start ingest to test, stop ingest temporarily while I tweaked things, and restart ingest later and it would resume ingesting at the end of my dataset.

I understand the utility in providing a start time, but we're throwing away other utility for it with this change.  Perhaps just adding a convenient option to make the start time "now" would be sufficient.  Or perhaps defaulting this start time to the current time would work.